### ISSUE TITLE
Forgot/Reset password functionality on the login screen

### ISSUE THREAD

**Issue by burningice2866** [Orckestra/C1-CMS-Foundation#492]
Adding a Forgot / Reset password function on the login screen for users with an email address would help a great deal on bigger sites with lots of less tech-savy editors who are otherwise contacting their it department each time they forget their password. Which surprisingly for some editors happens a lot :)

**Comment by portal7** [comment]
This feature and Admin Password Recovery Method "Simply" or well documented.... will be so Cool!  
👍  💯 🥇  (and everything that help!  hehe)

**Comment by ai-fwd** [comment]
I agree this is needed but the only issue i see is that usernames don't have to be email addresses so not sure how it would work -- likely some way to plugin some "lookup" logic in case the username isn't an email.

**Comment by DBailey635** [comment]
If using the password recovery by email method, you'd also need to check if an SMTP server is configured in IIS (#147).
